### ISSUE TITLE
`recovery-reboot` doesn't reboot

### ISSUE THREAD

**Issue by umbernhard** [osresearch/safeboot#110]
Following the instructions in the TL;DR for an HP Elitebook 840 G6, Ubuntu 20.04. Everything goes smoothly up to the `recovery-reboot` step in the first block. `BootNext` seems to get set properly, but it prints 

```mount: /: mount point is busy.```

and silently exits. I think rebooting manually still works okay, but I'm not sure if this will cause issues later on in the process.

**Comment by umbernhard** [comment]
After some more digging, it reboots into _a_ recovery mode, but not safeboot recovery. I'm not sure if this has to do with my not signing the recovery kernel correctly or not, so I'll keep digging.

**Comment by sourceXORapprentice** [comment]
recovery-reboot has had the same "mount point is busy" error message on both HP Elitedesk 705 G3 and Dell Optiplex 5050. It seems like it isn't able to modify the boot entries. I have to manually reboot and modify my boot entries at this step too.

**Comment by umbernhard** [comment]
@sourceXORapprentice you mean edit the entries by running `efibootmgr`? I haven't had issues with that, just that recovery-reboot fails due to drives being mounted/unmounted.

**Comment by sourceXORapprentice** [comment]
Correct, sorry muddled issues mentioning a separate issue with `BootNext`, after fiddling more with firmware settings it seems it now allows it to set on the Dell.

In all cases I've also received:
`mount: /: mount point is busy.`
